### PR TITLE
PLENTY-310 show number of selected filter values

### PR DESCRIPTION
**Pull request by RimantasAdeoWeb**

### CODE REVIEW

File: resources/js/src/app/components/itemList/filter/ItemFilter.js [hunk 1]
```diff
@@ -37,6 +37,18 @@ Vue.component("findologic-item-filter", {
             selectedFacets: state => state.itemList.selectedFacets,
             isLoading: state => state.itemList.isLoading
         }),
+
+        selectedValuesCount() {
```
Comment (TheKeymaster) [inline resources/js/src/app/components/itemList/filter/ItemFilter.js:41]: I prefer having this as a computed property.

File: resources/js/src/app/components/itemList/filter/ItemFilter.js [hunk 2]
```diff
@@ -37,6 +37,18 @@ Vue.component("findologic-item-filter", {
...
+            let count = 0;
+
+            this.facet.values.forEach(function(value) {
```
Comment (TheKeymaster) [inline resources/js/src/app/components/itemList/filter/ItemFilter.js:44]: How about this? It should yield the same result (I didn't test it), but is easier to read in my opinion.

```js
const selectedFacets = this.facet.values.filter(function (value) {
    return value.selected;
});

return selectedFacets.length;
```

Comment (TheKeymaster) [reply resources/js/src/app/components/itemList/filter/ItemFilter.js:44]: Also don't we already have a helper `getSelectedFilters` for this purpose, why not reuse the existing logic? This way we already could utilize `getSelectedFilters().length` in the template.

**Review by TheKeymaster — CHANGES_REQUESTED** [review]

File: resources/js/src/app/components/itemList/filter/ItemFilter.js (hunk 1 above)
Comment (RimantasAdeoWeb) [reply resources/js/src/app/components/itemList/filter/ItemFilter.js:41]: But it IS a computed property. Or am I misunderstanding something?

File: resources/js/src/app/components/itemList/filter/ItemFilter.js (hunk 2 above)
Comment (RimantasAdeoWeb) [reply resources/js/src/app/components/itemList/filter/ItemFilter.js:44]: getSelectedFilters returns the selected filters, not the selected values for each filter. So extra stuff would have to be done which would end up being messier than this, unfortunately.

Comment (TheKeymaster) [reply resources/js/src/app/components/itemList/filter/ItemFilter.js]: Ah I see.

File: resources/js/src/app/components/itemList/filter/ItemFilter.js (hunk 1 above)
Comment (TheKeymaster) [reply resources/js/src/app/components/itemList/filter/ItemFilter.js:41]: Oh, it already is. Then forget this :)

**Review by TheKeymaster — APPROVED** [review]

**Pull request merged**